src/app/public/modules/flyout/flyout.service.ts

```diff
@@ -47,6 +47,11 @@ import {
   SkyFlyoutMessageType
 } from './types/flyout-message-type';
 
+/**
+ * The `SkyFlyoutService` launches flyouts and provides a common look and feel.
```

Comment:
For consistency with how we handle descriptions of directives, services, etc, I'd delete the first two words and just start with `Launches flyouts and...`

Reply:
Done.